api/services/ocean/onpremise/paths/cluster.yaml

```diff
@@ -9,6 +9,14 @@ delete:
   parameters:
     - $ref: "../../../../commons/parameters/accountId.yaml"
     - $ref: "../../commons/parameters/oceanId.yaml"
+    - in: "path"
```

Comment:
@vlad666spot  why changing?

Reply:
![Screenshot 2020-11-05 at 11 10 05](https://user-images.githubusercontent.com/64230439/98220793-85a91a80-1f57-11eb-9097-dcb800353801.png)
here in path we have not OceanId, but ClusterId, so it was the reason of change